src/Dapr.Actors.Generators/Templates.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Dapr.Actors.Generators
+{
+    /// <summary>
+    /// Templates for generating source code.
+    /// </summary>
+    internal static partial class Templates
+    {
+        /// <summary>
+        /// Returns the <see cref="SourceText"/> for the ActorMethodAttribute.
+        /// </summary>
+        /// <param name="destinationNamespace">Namespace where to generate attribute.</param>
+        /// <returns>The <see cref="SourceText"/> representing the ActorMethodAttribute.</returns>
+        /// <exception cref="ArgumentNullException">Throws when destinationNamespace is null.</exception>
+        public static SourceText ActorMethodAttributeSourceText(string destinationNamespace)
+        {
+            if (destinationNamespace == null)
```

Comment:
I am not going to dictate here, but I do prefer to use `is null` rather than `== null` checks in my code. This is because the `==` can be overwritten to behave differently. It's highly unlikely to happen, but it can. You can read up about it [on this StackOverflow thread](https://stackoverflow.com/questions/40676426/what-is-the-difference-between-x-is-null-and-x-null) if you'd like. 

If that's not your style, I'm happy for you to resolve this comment with a simple comment saying something like "I'll stick to `== null` in my code".

Reply:
Handled, thank